flink-formats/flink-json/pom.xml

```diff
@@ -112,4 +112,25 @@ under the License.
 			</build>
 		</profile>
 	</profiles>
+
+	<build>
+		<plugins>
+			<!-- Enable enforcer plugin for checking dependency convergence -->
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-enforcer-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>dependency-convergence</id>
+						<goals>
+							<goal>enforce</goal>
+						</goals>
+						<configuration>
+							<skip>false</skip>
```

Comment:
shouldn't be necessary for flink-json. we need it in flink-table since we disable the enforcer in flink-libraries.